apps/api/src/app/admin/admin.service.ts

```diff
@@ -332,12 +333,18 @@ export class AdminService {
     scraperConfiguration,
     sectors,
     symbol,
-    symbolMapping
+    symbolMapping,
+    url
   }: Prisma.SymbolProfileUpdateInput & UniqueAsset) {
+    const symbolProfileOverride = {
+      name: name as string,
```

Comment:
Can you move down to keep the alphabetical order?

Reply:
Done